### ISSUE TITLE
Fetch images via URL

### ISSUE THREAD

**Issue by mkelley** [Small-Bodies-Node/sbn-survey-image-service#1]
Data is currently assumed to be accessible via the local file system.  However, we need to be able to instead fetch images via URL, make the cutout, and serve the result to the user.  To that end, I propose adding the following options to the data harvester, `--base-url` and `--strip-leading`.  For example, if the data live in sub-directories of `/data/neat/tricam`:

```
/data/neat/tricam/data/p20020701/obsdata
/data/neat/tricam/data/p20020702/obsdata
/data/neat/tricam/data/p20020703/obsdata
/data/neat/tricam/data/p20020704/obsdata
```

and are served via the URLs of the form: `https://sbnarchive.psi.edu/pds3/neat/tricam/data/p20020701/obsdata/filename` etc., then the command-line parameters could be:

`...add /data/neat/tricam/data/p*/obsdata --base-url=https://sbnarchive.psi.edu/pds3/ --strip-leading=/data/ `

In other words, the data harvester (`add`) would be run on a machine with local file access to the archival data set, `/data/` would be stripped from the start of each file's path, appended to the base-url, and the result inserted into the database.

@jstone-psi do you think this will work out on your end?

**Comment by jstone-psi** [comment]
@mkelley That seems reasonable. Those options seem similar to the PDS Harvest Tool, so it's a model that everyone who uses it should be familiar with.

**Comment by mkelley** [comment]
Implemented!
